README.md

```diff
@@ -195,14 +195,26 @@ altTextAndImages :: Scraper String [(String, URL)]
 altTextAndImages =
     -- 1. First narrow the current context to each img tag.
     chroots "img" $ do
-        -- 2. Use Any to access all the relevant content from the the currently
+        -- 2. Use anySelector to access all the relevant content from the the currently
         -- selected img tag.
-        altText <- attr "alt" Any
-        srcUrl  <- attr "src" Any
+        altText <- attr "alt" anySelector
+        srcUrl  <- attr "src" anySelector
         -- 3. Combine the retrieved content into the desired final result.
         return (altText, srcUrl)
 ```
 
 For the full source of this example, see
 [generalized-repetition](https://github.com/fimad/scalpel/tree/master/examples/generalized-repetition/)
 in the examples directory.
+
+### Matching wildcards
+
+ - `anySelector` will match all tags
+ - `AnyTag` will match all tag names
+ - `AnyAttribute` will match all tag attributes
+
+### OverloadedStrings
```

Comment:
Lets move this to be the first section under Tips & Tricks, I think this is probably the most widely useful thing listed here currently.